test/integration/shoots/logging/seed_logging_stack.go

```diff
@@ -55,34 +58,39 @@ const (
 	garden                        = "garden"
 	loggerDeploymentName          = "logger"
 	logger                        = "logger-.*"
-	fluentBitConfigMapName        = "fluent-bit-config"
 	fluentBitClusterRoleName      = "fluent-bit-read"
 	simulatesShootNamespacePrefix = "shoot--logging--test-"
-	lokiConfigMapName             = "loki-config"
 )
 
 var _ = ginkgo.Describe("Seed logging testing", func() {
-
-	f := framework.NewShootFramework(nil)
-	gardenNamespace := &corev1.Namespace{}
-	fluentBit := &appsv1.DaemonSet{}
-	fluentBitConfMap := &corev1.ConfigMap{}
-	fluentBitService := &corev1.Service{}
-	fluentBitClusterRole := &rbacv1.ClusterRole{}
-	fluentBitClusterRoleBinding := &rbacv1.ClusterRoleBinding{}
-	fluentBitServiceAccount := &corev1.ServiceAccount{}
-	fluentBitPriorityClass := &schedulingv1.PriorityClass{}
-	clusterCRD := &apiextensionsv1.CustomResourceDefinition{}
-	lokiSts := &appsv1.StatefulSet{}
-	lokiServiceAccount := &corev1.ServiceAccount{}
-	lokiService := &corev1.Service{}
-	lokiConfMap := &corev1.ConfigMap{}
-	lokiPriorityClass := &schedulingv1.PriorityClass{}
+	var (
+		f                           = framework.NewShootFramework(nil)
+		gardenNamespace             = &corev1.Namespace{}
+		fluentBit                   = &appsv1.DaemonSet{}
+		fluentBitConfMap            = &corev1.ConfigMap{}
+		fluentBitService            = &corev1.Service{}
+		fluentBitClusterRole        = &rbacv1.ClusterRole{}
+		fluentBitClusterRoleBinding = &rbacv1.ClusterRoleBinding{}
+		fluentBitServiceAccount     = &corev1.ServiceAccount{}
+		fluentBitPriorityClass      = &schedulingv1.PriorityClass{}
+		clusterCRD                  = &apiextensionsv1.CustomResourceDefinition{}
+		lokiSts                     = &appsv1.StatefulSet{}
+		lokiServiceAccount          = &corev1.ServiceAccount{}
+		lokiService                 = &corev1.Service{}
+		lokiConfMap                 = &corev1.ConfigMap{}
+		lokiPriorityClass           = &schedulingv1.PriorityClass{}
+		shootAsSeedClient           kubernetes.Interface
+	)
 
 	framework.CBeforeEach(func(ctx context.Context) {
+		var err error
 		checkRequiredResources(ctx, f.SeedClient)
+		shootAsSeedClient, err = kubernetes.NewClientFromSecret(ctx, f.SeedClient.Client(), framework.ComputeTechnicalID(f.Project.Name, f.Shoot), gardencorev1beta1.GardenerName, kubernetes.WithClientOptions(client.Options{
```

Comment:
What is a "shoot as seed" client? It's highly confusing, can we find a more appropriate name?

Reply:
I am using a simple shoot to simulate seed with Cluster resources and shallow control plane namespaces.
So in this case I am using the testing shoot as a fake seed.
Do you have any suggestions for a name?
Maybe `shootAsFakeSeed` or `shootWithSeedSchemaClient` ?

Reply:
I see the challenge. Maybe just `shootClient` because that's what it is? You could add a comment in the above line explaining that this shoot is used as seed for this integration test.

Reply:
Ok, this sounds good. Thanks